utils/build-script-impl

```diff
@@ -712,6 +712,18 @@ function set_build_options_for_host() {
           -DLLVM_TOOL_COMPILER_RT_BUILD:BOOL="$(false_true ${SKIP_BUILD_COMPILER_RT})"
           -DLLVM_BUILD_EXTERNAL_COMPILER_RT:BOOL="$(false_true ${SKIP_BUILD_COMPILER_RT})"
       )
+      # Android doesn't support building all of compiler-rt yet.
+      if [[ "${host}" == "android-"* ]]; then
```

Comment:
I would recommend not starting creating more checks with `if`, when there's a `case` block above. Since you want to apply this to every Android, maybe you need to do like the Darwin targets do, and have a first check for `android-*)`, and then a second case for the specific architectures.

Something like:

```
# Around line 454
case "${host}" in
    android-*)
        llvm_cmake_options+=(
          -DDEFAULT_SYSROOT:PATH="$(dirname ${PREFIX})"
        )
        # Android doesn't support building all of compiler-rt yet
        if [[ ! $(is_cross_tools_host "${host}" ]] ; then
          llvm_cmake_options+=(
              -DCOMPILER_RT_INCLUDE_TESTS:BOOL=FALSE
          )
        fi
        case "${host}" in
            android-aarch64)
                # The code that already existed here and the rest of the Android cases.
        esac
        ;;
    # The other cases here...
```

Reply:
Will do.

Reply:
Made the changes you asked for.